src/kirby/components/page/page.component.ts

```diff
@@ -175,13 +177,18 @@ export class PageComponent implements OnInit, OnDestroy, AfterViewInit {
         this.onEnter();
       }
     });
+
+    addEventListener(tabClicked, () => {
+      this.content.scrollToTop(KirbyAnimation.Duration.LONG);
+    });
   }
 
   ngOnDestroy(): void {
     if (this.routerEventsSubscription) {
       this.routerEventsSubscription.unsubscribe();
     }
     this.pageTitleIntersectionObserverRef.disconnect();
+    removeEventListener(tabClicked, null);
```

Comment:
What does this method refer to? If it's window.removeEventListener it should be explicit...